### PR TITLE
Glimpse: Add utils to build queries

### PR DESCRIPTION
**Pull request by luca020400**
Change-Id: I0e58cb4179c0e7852275d74f501d40b27a68dae1

### CODE REVIEW

**Comment by luca020400** [comment]
How do you feel about something like this?

**Comment by SebaUbuntu** [comment]
Looks nice

File: app/src/main/java/org/lineageos/glimpse/query/Query.kt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package org.lineageos.glimpse.query
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/glimpse/query/Query.kt]: copyright

File: app/src/main/java/org/lineageos/glimpse/query/Query.kt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                or(
+                    eq("test", 1),
+                    and(
+                        eq("", ""), eq("", "")
+                    )
+                )
+            )
+        }
+    }
+}
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/glimpse/query/Query.kt]: newline

File: app/src/main/java/org/lineageos/glimpse/fragments/AlbumFragment.kt [hunk 3]
```diff
@@ -138,44 +139,47 @@ class AlbumFragment : Fragment(R.layout.fragment_album), LoaderManager.LoaderCal
...
-                        append(" = ?")
+                        eq(MediaStore.Files.FileColumns.BUCKET_ID, "?")
                     }
                 }
             }
+
+            val selection = buildQuery {
+                and(imageOrVideo, albumKind)
+            }
+
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/glimpse/fragments/AlbumFragment.kt]: one newline less

**Pull request merged**